src/main/scala/com/amazon/deequ/analyzers/jdbc/JdbcAnalyzer.scala

```diff
@@ -193,4 +193,13 @@ object Preconditions {
         s"but found ${JDBCType.valueOf(columnDataType).getName} instead!")
     }
   }
+
+  /** Statement has no ; outside of quotation marks (SQL injections)*/
```

Comment:
this is not conform with styleguides
```suggestion
  /* Statement has no ; outside of quotation marks (SQL injections)*/
```